discovery-provider/src/challenges/challenges.json

```diff
@@ -2,66 +2,75 @@
 	{
 		"id": "profile-completion",
 		"type": "numeric",
-		"amount": "5",
+		"amount": "1",
 		"active": true,
-		"step_count": 7
+		"step_count": 7,
+		"starting_block": 25346436
 	},
 	{
 		"id": "listen-streak",
 		"type": "numeric",
-		"amount": "5",
-		"active": false,
-		"step_count": 7
+		"amount": "1",
+		"active": true,
+		"step_count": 7,
+		"starting_block": 116023891
```

Comment:
let's keep a close eye on this, but good thing is that this will be a subtle thing and it doesn't matter a whole lot when "activation" is